packages/cli/lang/en.lyaml

```diff
@@ -1028,6 +1028,7 @@ en:
         componentCount: "Found {{ numComponents }} components in this project"
         successStatusText: "DONE"
         failedStatusText: "FAILED"
+        errorFetchingTaskStatus: "Error fetching task status"
```

Comment:
I'm not the best at error messages, and I'm still learning my way around this project, so I'm all 👂s if anyone has a better idea for this message.

Reply:
If we do end up using this error message, could be good to specify "build" or "deploy" since task doesn't necessarily mean anything to users

Reply:
Good point, it looks like the `i18n` module supports interpolation and I can make this more specific